view/perspective/app.js

```diff
@@ -170,7 +170,10 @@ function setupSocketIOClient(persBody) {
     socket = _io.connect(`${_realtimeApplication}${namespace}`, options)
              .on('connect', function() {
                this.emit('auth', _userSession);
-             });
+             })
+             .on('auth error', (err) =>
```

Comment:
any tests for these here? (mostly I'm curious about whether socket.io supports spaces in the eventName)

Reply:
This is tested on the realtime side, in the token auth PR.